### PR TITLE
Completed Toast and Image

### PR DESCRIPTION
**Pull request by varun1233**
Completed Toast and Image

### CODE REVIEW

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.responsive-image {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
```
Comment (rahulk31) [inline components/image/image.css]: The image is not responsive if the height of the img-container increases. Need to work on this.

**Review by rahulk31 — COMMENTED** [review]
Overall looks good to me.

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
You should change the height of the code snippet, rest looks good

**Comment by varun1233** [comment]
> You should change the height of the code snippet, rest looks good

I have noted this feedback, I will be refactoring the prism js code once library is completed, will work on this then

File: components/image/image.css (hunk 1 above)
Comment (varun1233) [reply components/image/image.css]: Done

**Review by rahulk31 — APPROVED** [review]
Looks good to me, Merge ready 👍

File: components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
+.strike-through {
+  text-decoration: line-through;
+}
+.discount-percentage {
+  color: var(--primary-color);
+}
+.card-box-shadown {
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```
Comment (zoef-yt) [inline components/cards/cards.css:8]: Use colour

File: components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+}
+.margin-bottom-1 {
+  margin-bottom: 1rem;
+}
+.margin-top-1 {
+  margin-top: 1rem;
+}
+
```
Comment (zoef-yt) [inline components/cards/cards.css:16]: Remove spacing

File: components/cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+.card-horizontal {
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  max-width: 60%;
+  border: 1px solid var(--primary-background-color);
+}
+.card-vertical:hover,
+.card-horizontal:hover {
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
```
Comment (zoef-yt) [inline components/cards/cards.css:34]: Use var

File: components/utility.css [hunk 5]
```diff
@@ -19,3 +19,47 @@ img {
   border-radius: 33px;
   background-color: #0e0f0f5d;
```
Comment (zoef-yt) [inline components/utility.css:20]: Use var

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/utility.css (hunk 5 above)
Comment (varun1233) [reply components/utility.css:20]: Done

File: components/cards/cards.css (hunk 4 above)
Comment (varun1233) [reply components/cards/cards.css:34]: Done

File: components/cards/cards.css (hunk 3 above)
Comment (varun1233) [reply components/cards/cards.css:16]: Done

File: components/cards/cards.css (hunk 2 above)
Comment (varun1233) [reply components/cards/cards.css:8]: done

**Pull request merged**